packages/foundry-sdk-generator/package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "@osdk/api": "workspace:~",
     "@osdk/client": "workspace:~",
+    "@osdk/client.api": "workspace:~",
```

Comment:
Added this so that the generated packages from tests would correctly populate client.api version in the package.jsons